ajax/libs/react-native-elements/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "react-native-elements",
+  "version": "0.11.2",
+  "description": "React Native Elements & UI Toolkit",
+  "filename": "bundle.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/dabit3/React-Native-Elements.git"
```

Comment:
I use `git clone ssh://git@github.com/dabit3/React-Native-Elements.git` and it works.
But I have the question as same with you.